README.md

```diff
@@ -65,12 +71,10 @@ You can configure the retention policy by setting `pe_databases::backup::retenti
 
 ### Disable Maintenance
 
-The maintenance cron jobs will perform a `VACUUM FULL` on various `pe-puppetdb` tables to keep them lean and fast.
-A `VACUUM FULL` is a blocking operation and you will see the PuppetDB command queue grow while the cron jobs run.
-The blocking should be short lived and the PuppetDB command queue should work itself down after, however, if for some reason you experience issues you can disable the maintenance cron jobs.
+The maintenance SystemD timers will perform a `pg_repack` on various `pe-puppetdb` tables to keep them lean and fast.
+A `pg_repack` is a non-blocking maintence action, however, if for some reason you experience issues you can disable the maintenance SystemD timers,
 You can do so by setting `pe_databases::maintenance::disable_maintenance: true` in your hieradata.
```

Comment:
This is a parameter for a class that doesn't exist. @m0dular We need to add this parameter to the main class and check for both.